### DISCUSSION TITLE
Installing on v3 Camera

### DISCUSSION THREAD

**Discussion by madcar86** [gtxaspec/wz_mini_hacks#608] [General]
I am attempting to install it on a v3, but after putting the loaded SD Card into the camera, I can't ssh into the camera to confirm it's been installed. The Wyze app says it's on firmware 4.36.9.139. Am I missing anything or is there another way to confirm its installed?

`#wz_mini configuration

#### W E B CAMERA###
##THIS MODE DISABLES EVERYTHING AND IT WILL
## WORK AS A WEB CAMERA FOR YOUR PC ***ONLY***
WEB_CAM_ENABLE="false"
WEB_CAM_BIT_RATE="8000"
WEB_CAM_FPS_RATE="25"

#####NETWORKING#####
CUSTOM_HOSTNAME="Workshop"

ENABLE_USB_ETH="false"
ENABLE_USB_ETH_MODULE_AUTODETECT="true"
ENABLE_USB_ETH_MODULE_MANUAL=""
USB_ETH_MAC_ADDR=""

ENABLE_USB_DIRECT="false"
USB_DIRECT_MAC_ADDR="02:01:02:03:04:08"

ENABLE_USB_RNDIS="false"

ENABLE_IPV6="false"

ENABLE_WIREGUARD="false"
WIREGUARD_IPV4=""
WIREGUARD_PEER_ENDPOINT=""
WIREGUARD_PEER_PUBLIC_KEY=""
WIREGUARD_PEER_ALLOWED_IPS=""
WIREGUARD_PEER_KEEP_ALIVE=""

ENABLE_IPTABLES="false"

ENABLE_NFSv4="false"

ENABLE_RTL8189FS_DRIVER="true"
ENABLE_ATBM603X_DRIVER="true"

NTP_SERVER=""

##### NETWORK INTERFACE BONDING #####
BONDING_ENABLED="false"
BONDING_PRIMARY_INTERFACE="eth0"
BONDING_SECONDARY_INTERFACE="wlan0"
BONDING_LINK_MONITORING_FREQ_MS="100"
BONDING_DOWN_DELAY_MS="5000"
BONDING_UP_DELAY_MS="5000"
BONDING_FAIL_OVER_MAC="0"

#####ACCESSORIES#####
REMOTE_SPOTLIGHT="false"

#####VIDEO STREAM#####
RTSP_LOGIN="madcar86"
RTSP_PASSWORD="william86"
RTSP_PORT="8554"

RTSP_HI_RES_ENABLED="true"
RTSP_HI_RES_ENABLE_AUDIO="true"

RTSP_LOW_RES_ENABLED="false"
RTSP_LOW_RES_ENABLE_AUDIO="false"

RTSP_AUTH_DISABLE="false"

VIDEO_0_MAX_BITRATE=""
VIDEO_0_TARGET_BITRATE=""
VIDEO_0_ENC_PARAMETER=""
VIDEO_0_FPS=""

VIDEO_1_MAX_BITRATE=""
VIDEO_1_TARGET_BITRATE=""
VIDEO_1_ENC_PARAMETER=""
VIDEO_1_FPS=""

GO2RTC_SERVER_ENABLED="false"
GO2RTC_HI_RES_ENABLED="false"
GO2RTC_HI_RES_ENABLE_AUDIO="false"
GO2RTC_LOW_RES_ENABLED="false"
GO2RTC_LOW_RES_ENABLE_AUDIO="false"

#####RTMP STREAMING#####
### AND RTSP AUDIO ###
RTMP_STREAM_ENABLED="false"
RTMP_STREAM_FEED="video1_unicast"
RTMP_STREAM_SERVICE="youtube"
RTMP_STREAM_DISABLE_AUDIO="false"
RTMP_STREAM_YOUTUBE_KEY=""
RTMP_STREAM_TWITCH_KEY=""
RTMP_STREAM_FACEBOOK_KEY=""

#####GENERAL#####
LIBCALLBACK_ENABLE="true"
ENABLE_SWAP="true"
ENABLE_USB_STORAGE="false"
ENABLE_EXT4="false"
ENABLE_CIFS="false"
DISABLE_FW_UPGRADE="true"
AUDIO_PROMPT_VOLUME="50"
ENABLE_MP4_WRITE="false"
NIGHT_DROP_DISABLE="true"
DISABLE_MOTOR="false"
ENABLE_FSCK_ON_BOOT="false"
ENABLE_CAR_DRIVER="false"
ENABLE_LOCAL_DNS="false"
ENABLE_CRONTAB="false"
ENABLE_SELFHOSTED_MODE="false"

#####DEBUG#####
#drops you to a shell via serial, doesn't load app_init.sh
DEBUG_ENABLED="false"
DEBUG_INITRAMFS_ENABLED="false"
DEBUG_PASSWORD="false"
ENABLE_SYSLOG_SAVE="false"

####WEB####
WEB_SERVER_ENABLED="true"
WEB_SERVER_AUTH="true"
WEB_SERVER_LOGIN="madcar86"
WEB_SERVER_PASSWORD="william86"

#####SCRIPTING#####
CUSTOM_SCRIPT_PATH=""
`

**Comment by bjs-pdx** [comment]
What is on the SD card?  Is it formatted fat32?  Did you power cycle the camera after inserting the sd card?  Not much to go on here.
If you rebooted with the sd card in the camera you should have heard an audible message about extracting the wz_mini swap.  If you didn't hear that, it isn't running.  See also https://github.com/gtxaspec/wz_mini_hacks/discussions/604

**Reply by madcar86** [reply]
It's formatted fat32 with the raspberry pi imager, and it has the contents of the SD_Root folder from the repo. I also add the SSH Public Key to the file. I unplug the camera, install the card, and plug it back in. I don't get any audible messages, it just boots up as normal. Do I need to hold down the setup button?

**Reply by Bob-AOL** [reply]
Wz_mini doesn't do anything beyond the ordinary Wyze OEM app unless you tell it to. Otherwise the camera just behaves as a normal v3.
BTW, the app is your friend when needing to check device details.

Your current .conf file isn't going to do anything except provide access to the web interface. SSH may or may not be functional and can be a bag o' snakes on a good day if not configured as described in the README..., a very basic implementation at that. It is also not a requirement for basic wz_mini functionality.

And depending on how you edited and saved the .conf file, text formatting could possibly be a problem as well as the formatting of the SD card itself.

The wz_mini.conf file as currently configured should at least start the web server if the bootloader loaded wz_mini. Just enter the camera's IP address into a browser and see if the server responds. The latest version looks to have an authentication front end now and should presumably initiate a login screen if the server is active. (Just guessing there as I've not tried the latest distro yet.)

Otherwise, back to the drawing board -- starting with a very careful review of the README to ensure you are covering the bases correctly and not over-complicating anything. The simplest and most efficient implementation out of the box is actually the Windows approach. It only works one way and is butt simple. Too many ways to outsmart one's self with one's own self-exalted Linux command line prowess if one doesn't stay within the conventional basics provided in the README.

Follow directions. Keep it simple for starters. It will work.

**Comment by madcar86** [comment]
my main goal is to just have RTSP access. I was following a guide on YouTube. What configs do I need to change? I tried to go to the IP address but it doesn't load the webserver and my NVR doesn't find the camera either. I never heard the audio saying it was extracting the files. 

I went through the Setup/Install on the wiki and I thought I followed everything. Get the camera on 4.46.9.139, edit the config file, and copy the SD folder to FAT32 formatted card. Insert into camera and power on. No audio confirmation or web server. What am I missing

**Reply by bjs-pdx** [reply]
First question:  can you access the camera via the Wyze App?  If so, have you verified the IP address?  If you can't access the web server or ssh or ping the camera's IP, it isn't on the network.  If you didn't hear any audio about extracting the swap, the configuration isn't working.  Try a different SD card and start over.

**Reply by madcar86** [reply]
Yes I can verify the IP on the Wyze App and the router. But nothing on no webserver or ssh. I'll try a new card.

**Comment by madcar86** [comment]
It was the SD Card, for whatever reason the camera wasn't reading it. Works fine with another SD Card. Thanks for everyone's help.
